src/coreclr/minipal/Windows/dn-u16.cpp

```diff
@@ -0,0 +1,75 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include <Windows.h>
+#include <wchar.h>
+
+#include <dn-u16.h>
+
+size_t strlen_u16(const WCHAR* str)
+{
+    if (str == NULL)
```

Comment:
Why do we need this null check? The standard Windows wcslen does not have it, so we should not need it - no code can assume this null check today.